Tests/EmailValidator/Validation/MultipleValidationWitAndTest.php

```diff
@@ -32,6 +32,19 @@ public function testEmptyListIsNotAllowed()
         new MultipleValidationWithAnd([]);
     }
 
+    public function testValidationIsValid()
+    {
+        $lexer = $this->getMock("Egulias\\EmailValidator\\EmailLexer");
```

Comment:
Exactly. but I'm gonna use existing CS in here.
However, `::class` should be used in whole of PJ because we are using PHP 5.5 or later.